src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/CPP/DWriteWrapper/Factory.h

```diff
@@ -25,159 +25,18 @@ namespace MS { namespace Internal { namespace Text { namespace TextInterface
     /// <summary>
     /// The root factory interface for all DWrite objects.
     /// </summary>
-    private ref class Factory sealed : public CriticalHandle
+    private ref class InternalFactory abstract sealed
```

Comment:
I renamed `Factory` to `InternalFactory` to avoid conflicts in PresentationCore since I added a new `Factory` class with the same namespace.